app/views/assignments/find_client.html.slim

```diff
@@ -22,7 +20,57 @@ h3= t_attr(:single_accompaniment, Volunteer)
 
 = render 'availability/show', available: @volunteer
 
-.row
-  .col-xs-12
-    h1= t('clients_need_accompanying')
-= render 'clients/columns', clients: @need_accompanying, volunteer: @volunteer
+h1= t('clients_need_accompanying')
+
+nav.navbar.section-navigation
+  ul.list-inline
+    li= render 'gender_request_select'
+    li= render 'age_request_select'
+    li= render 'language_skills_language_select'
+
+.text-center
+  = will_paginate @need_accompanying, renderer: BootstrapPagination::Rails
```

Comment:
We could add a helper (`paginate` or maybe `bootstrap_paginate`) to avoid having to specify the `renderer:` every time.

Reply:
Would actually be out of scope, but as it is too good to be not done, I did it.